contracts/oracles/TwapOracle.sol

```diff
@@ -63,6 +56,12 @@ contract TwapOracle is OwnableUpgradeable, TwapInterface {
         uint256 newAcc
     );
 
+    ///@notice Keeps track of observation of token mapped by address , update on evry updateTwap invocation.
+    mapping(address => Observation[]) public observations;
+
+    ///@notice index of array which probably falls in current anchor period mapped with asset address
+    mapping(address => uint256) public windowStart;
```

Comment:
Also this can be made `private`?

Reply:
Same comment as above for observations mapping.